apps/swirl-docs/src/components/IconGrid/IconGridItem.tsx

```diff
@@ -0,0 +1,41 @@
+import { FlipPopover } from "@getflip/swirl-components-react";
+import { FunctionComponent } from "react";
+import { IconsMetaData } from "src/pages/icons";
+import IconInfo from "./IconInfo";
+
+interface IconGridProps {
+  id: string;
+  icon: string;
+  icons: IconsMetaData;
+  handleTileClick: (iconName: string) => void;
+}
+
+const IconGridItem: FunctionComponent<IconGridProps> = ({
+  id,
+  icon,
+  icons,
+  handleTileClick,
+}) => {
+  return (
+    <li>
+      <a
+        id={id}
+        href={`#${icons[icon]?.name}`}
+        onClick={() => handleTileClick(icons[icon]?.name)}
+        className="flex flex-col justify-center items-center py-4 border-1 rounded-lg"
+      >
+        <i
+          className={`swirl-icons-${icons[icon]?.name}28 text-icon-strong`}
+        ></i>
+        <span className="text-text-subdued">{icons[icon]?.name}</span>
+      </a>
+      <FlipPopover label="Icon Info" popoverId={`popover-${id}`} trigger={id}>
```

Comment:
**question:** The design is still WIP and out of scope of this PR, right? Spacings and the formatting of the sample code are a bit off. As well as details in the button design.

Reply:
Yes i am collecting stuff like that and will implement it in [this ticket](https://flipapp.myjetbrains.com/youtrack/issue/APP-11929/Implement-Figma-Design-for-getflipdev)